### PR TITLE
#35: o.l.c.m.MatchesRegex

### PR DESCRIPTION
**Pull request by dgroup**
#35

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#57](https://github.com/llorllale/cactoos-matchers/issues/57) is now in scope, role is `REV`

**Comment by dgroup** [comment]
@llorllale, could you please review the RR ?

File: src/test/java/org/llorllale/cactoos/matchers/MatchesRegexTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ * @since 1.0.0
+ */
+public final class MatchesRegexTest {
+
+    /**
+     * Example of {@link MatchesRegex} usage.
+     */
+    @Test
+    public void matchPositive() {
+        MatcherAssert.assertThat(
```
Comment (llorllale) [inline src/test/java/org/llorllale/cactoos/matchers/MatchesRegexTest.java]: @dgroup we should use `Assertion` and `Matches`

**Comment by llorllale** [comment]
@dgroup let's start using `Assertion` and `Matches` in all our PRs

File: src/test/java/org/llorllale/cactoos/matchers/MatchesRegexTest.java (hunk 1 above)
Comment (dgroup) [reply src/test/java/org/llorllale/cactoos/matchers/MatchesRegexTest.java]: @llorllale, done.

**Comment by dgroup** [comment]
@llorlale, agree.

**Comment by dgroup** [comment]
@llorllale, done, please review PR.

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:llorllale/cactoos-matchers#57` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#57](https://github.com/llorllale/cactoos-matchers/issues/57) is now out of scope

**Pull request merged**